Dockerfile

```
@@ -53,14 +54,17 @@ RUN nix develop --command bash -c ' \
     if [ "$BUILD_PROFILE" = "release" ]; then \
-        cargo build --release --bin server; \
+        cargo build --release --bin server && \
+        cargo build --release --bin reporter; \
     else \
-        cargo build --bin server; \
+        cargo build --bin server && \
+        cargo build --bin reporter; \
     fi'
 
 # Fix binary interpreter path to use standard Linux paths
 RUN apt-get update && apt-get install -y patchelf && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)